spec/lib/savings_account_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'spec_helper.rb'
+
+
+describe BankAccount::SavingsAccount do
+
+  context "#init" do
+
+    it "knows about SavingsAccount class" do
+      expect(BankAccount::SavingsAccount).to respond_to :new
+    end
+
+    it "doesn't initialize a savings account with < $10" do
+      expect{(BankAccount::SavingsAccount.new(3, 5))}.to raise_exception(ArgumentError)
+    end
+  
+  end
+
+  context "balance" do
+
+    let(:savings_account) {BankAccount::SavingsAccount.new(4, 150)}
+
+    it "withdraws the correct amount + $2" do
```

Comment:
Would also recommend not using specific $ amounts in these tests in case the fee ever changes.